core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -896,13 +938,13 @@ public String toString() {
   public static final PropertyKey WORKER_FILE_BUFFER_SIZE =
       new Builder(Name.WORKER_FILE_BUFFER_SIZE)
           .setDefaultValue("1MB")
+          .setDescription("When worker writing data, the size of initial buffer")
```

Comment:
The size of the initial buffer, when the worker writes data.

Is this when the worker is writing to a block? to a ufs file?

Reply:
fixed